microsoft/PowerToys
# [ShortcutGuide] Installer and pipeline

## Summary of the Pull Request

**What is this about:**

**What is include in the PR:** 

**How does someone test / validate:** 
- Build MSI and install it. Verify that SG works
- Pipeline will be tested later

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
